frontend/src/text_resources.js

```diff
@@ -782,85 +770,85 @@ let LOCALIZE = new LocalizedStrings({
         }
       },
       emails: {
+        email0: {
```

Comment:
even if we index by 0 in code, the user will always see it counted by 1.

Reply:
Fixed in 63cf407